Java/other/BenjaminBulb.java

```diff
@@ -0,0 +1,25 @@
+//Link to problem: https://leetcode.com/problems/bulb-switcher/
```

Comment:
Remove this file, its irrelevant to this PR.

Reply:
Okay sure. This got here by mistake.